js/queue/queue-data-structure.js

```diff
@@ -10,17 +10,51 @@ class QueueDataStructure {
 
   canEnqueue() {
     // ... your code goes here
+    if (this.queueControl <= this.MAX_SIZE) {
+      return true
+    } else {
+      return false
+    }
   }
 
   isEmpty() {
     // ... your code goes here
+    if (this.queueControl.length == 0) {
+      return true
+
+    } else {
+      return false
+    }
```

Comment:
Aquí sobra una linea porque si

```suggestion
    if (this.queueControl.length == 0) {
      return true
    } else {
      return false
    }
```